composables/httpd.ts

```diff
@@ -0,0 +1,22 @@
+const PER_PAGE = 100
+
+function createHttpdBaseURL(domainName = 'seed.radicle.xyz') {
+  return `https://${domainName}/api/v1`
```

Comment:
What if the given domain isn't under http**s**?

_hint hint_ ;)

![image](https://github.com/cytechmobile/radicle-planning-boards/assets/7542054/4cfb739d-75ae-4772-91dc-4fad270ac933)


What if the given `domainName` contains a trailing `/`?
 
Always use [`URL constructor`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) for safer URL parsing.

Reply:
Good tip, I'll keep it in mind.